sormas-backend/src/test/java/de/symeda/sormas/backend/contact/ContactFacadeEjbTest.java

```diff
@@ -1036,6 +1038,62 @@ public void testGetIndexListByARestrictedAccessToAssignedEntities() {
 		assertEquals(2, getContactFacade().getIndexList(null, 0, 100, null).size());
 	}
 
+	@Test
+	public void testGetIndexListWithLimitedDisease() {
+		RDCF rdcf1 = creator.createRDCF("Region1", "District1", "Community1", "Facility1", "PointOfEntry1");
+		RDCF rdcf2 = creator.createRDCF("Region2", "District2", "Community2", "Facility2", "PointOfEntry2");
+
+		final UserDto surveillanceOfficer = creator.createSurveillanceOfficer(rdcf1);
+
+		PersonDto contactPerson = creator.createPerson("Contact", "Person");
+		final ContactDto contact = creator.createContact(
+			surveillanceOfficer.toReference(),
+			null,
+			contactPerson.toReference(),
+			null,
+			new Date(),
+			new Date(),
+			Disease.CORONAVIRUS,
+			rdcf1);
+
+		PersonDto contactPerson2 = creator.createPerson("Contact2", "Person2");
+		final ContactDto contact2 = creator
+			.createContact(surveillanceOfficer.toReference(), null, contactPerson2.toReference(), null, new Date(), new Date(), Disease.EVD, rdcf1);
+
+		PersonDto contactPerson3 = creator.createPerson("Contact3", "Person3");
+		final ContactDto contact3 = creator.createContact(
+			surveillanceOfficer.toReference(),
+			null,
+			contactPerson3.toReference(),
+			null,
+			new Date(),
+			new Date(),
+			Disease.CORONAVIRUS,
+			rdcf2);
+
+		PersonDto contactPerson4 = creator.createPerson("Contact3", "Person3");
+		final ContactDto contact4 = creator
+			.createContact(surveillanceOfficer.toReference(), null, contactPerson4.toReference(), null, new Date(), new Date(), Disease.EVD, rdcf2);
+
+		loginWith(surveillanceOfficer);
```

Comment:
the asserts for a normal surveillance officer are good to go into a separate test

this test is called `testGetIndexListWithLimitedDisease` so please test only what it supposed to be tested

Reply:
removed the asserts that exist in other tests. Only those connected to limited disease are left